pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -36,6 +36,11 @@ type TidbMonitor struct {
 	Status TidbMonitorStatus `json:"status"`
 }
 
+type DMMonitorSpec struct {
+	Clusters    []TidbClusterRef `json:"clusters"`
```

Comment:
Is it correct to use `TidbClusterRef`?

Reply:
It only includes `name` and `namespace` in that structure, but it's better to define a common one named `ClusterRef`. @BinChenn

Reply:
`TidbClusterRef` struct just contains three string var: `Namespace`, `Name` and `ClusterDomain`, we do not need to define another struct for dm.

Reply:
Then, I think @DanielZhangQD's suggestion is proper. We should rename it to a more common name.

Reply:
Renamed.